client/app/domain/zone/record/delete/domain-zone-record-delete.html

```diff
@@ -4,10 +4,15 @@
          data-wizard-on-finish="deleteDnsEntry"
          data-wizard-title="i18n.domain_configuration_dns_entry_delete_title">
 
-        <div data-wizard-step>
-            <p data-ng-if="ctrl.massDelete" data-ng-bind-html="trpl('domain_configuration_dns_entry_mass_delete_step1_question', ctrl.entryToDelete.length)"></p>
+        <div data-wizard-step
+             data-wizard-step-valid="!ctrl.loading">
 
-            <div data-ng-if="!ctrl.massDelete">
+            <oui-loader data-ng-if="ctrl.loading"></oui-loader>
```

Comment:
Careful on the naming change on the v2 : http://master.ui-kit.ovh/#!/oui-angular/spinner
See https://github.com/ovh-ux/ovh-manager-web/pull/267